ChangeLog.d/PSA-test-suites-NOT-using-UID-0.txt

```diff
@@ -0,0 +1,7 @@
+Changes
```

Comment:
([Background](https://github.com/ARMmbed/mbedtls/pull/5510#discussion_r801074797)) Most test changes don't need a changelog entry. This one does, though, since it's fixing a user-reported bug. This should be a Bugfix entry since it's affecting users, and the text should focus on the user-visible consequence and mention the issue number.

```
Bugfix
   * Fix unit tests that used 0 as the file UID. This failed on some
     implementations of PSA ITS. Fixes #3838.
```